frameworks_test.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/kudobuilder/kudo/pkg/test"
+)
+
+// Launch the KUDO tests.
+// To add more tests, add them to TestDirs belowe.
+func TestKudoFrameworks(t *testing.T) {
+	harness := test.Harness{
+		T:         t,
+		TestDirs:  []string{
+			"./repository/zookeeper/tests/",
```

Comment:
I was picturing the tests unique for each version of a framework, not at the top level.  

./repository/zookeeper/0.1.0/tests <- Tests for FrameworkVersion 0.1.0
./repository/zookeeper/0.2.0/tests < - Tests for FV 0.2.0

Reply:
The repository format KEP says its `repository/zookeeper/tests/` (all versions) and `repository/zookeeper/tests/0.1.0`. I think I agree with you, but I’m still tossing around how I want that to work, so let’s keep it how we have it for now and I’ll come through and update it once we’ve figured it out on the KEP?

Reply:
SGTM